portal/src/main/webapp/js/src/cross-cancer-plotly-plots.js

```diff
@@ -211,7 +211,18 @@ var ccPlots = (function (Plotly, _, $) {
                                 study_ids = _.filter(study_ids, function(study_id) { return study_id.indexOf(_tmp_target_study_obj[0].id) === -1; });
                                 mrna_profiles = _.filter(mrna_profiles, function(mrna_profile) { return mrna_profile.indexOf(_tmp_target_study_obj[0].id) === -1; });
                             }
-
+    
+                            //exclude stomach
+                            var _tmp_target_study_obj = _.filter(study_meta, function(obj) { return obj.name.indexOf("Stomach") !== -1 });
```

Comment:
It seems to be a lot of code to exclude a study. Can we put all the code in one function to get a list of studies to be used for this analysis, eg. TCGA provisional studies excluding STAD and ESCA?